values.yaml

```diff
@@ -157,6 +157,8 @@ fleetscheduler:
       schedule: "0 9 * * *" # Run every day at 9am
       command:
       - /usr/sbin/fleet-scheduler
-      - inventory
       - --config
       - /etc/fleet-scheduler/config.yaml
+      - inventory
+      - --page-size
```

Comment:
can you leave a comment on what page-size is here - is that the number of servers to fetch in each query?

Reply:
Yea its the amount of servers to get per page.
https://github.com/metal-toolbox/fleet-scheduler/blob/cc7e077eb4e115080c1a523af360649c182cb06d/cmd/inventory.go#L33

Ill add a comment here for the helm chart as well.